client/src/Tickets/TicketModal.jsx

```diff
@@ -84,26 +86,78 @@ function Charger({ stateChange }) {
   );
 }
 
-const AddTicketModel = ({ update, data, stateChange, toggleUserModal, setToggleUserModal }) => {
+const TicketModal = ({ onCreate, onUpdate }) => {
```

Comment:
Look at the difference in props now for the modal... no data is going in, but there are callbacks for data to go out...

Reply:
Also notice that the same modal is being used for Add AND Edit (although Edit still needs some more work)...

Reply:
Yea they are the same. I can't remember why, but I got stuck trying to combine them and just decided to split it up.